packages/flutter/lib/src/rendering/error.dart

```diff
@@ -79,14 +81,15 @@ class RenderErrorBox extends RenderBox {
 
   /// The text style to use when painting [RenderErrorBox] objects.
   static ui.TextStyle textStyle = new ui.TextStyle(
-    color: const Color(0xFFFFFF00),
+    color: const Color(0xFFFFFF66),
```

Comment:
Yellow on red wasn't the most contrasty for readability. It's still mostly the same color a bit brighter.